variants/dreamcatcher/variant.h

```diff
@@ -0,0 +1,107 @@
+#undef I2C_SDA
+#undef I2C_SCL
+#define I2C_SDA 16 // I2C pins for this board
+#define I2C_SCL 17
+
+#define I2C_SDA1 45
```

Comment:
Do you need
#define WIRE_INTERFACES_COUNT 2
?

Reply:
no, if you don't define it it will auto-adjust. configuration.h line 221...
```

#ifndef WIRE_INTERFACES_COUNT
// Officially an NRF52 macro
// Repurposed cross-platform to identify devices using Wire1
#if defined(I2C_SDA1) || defined(PIN_WIRE1_SDA)
#define WIRE_INTERFACES_COUNT 2
#elif HAS_WIRE
#define WIRE_INTERFACES_COUNT 1
#endif
#endif
```